src/media/writer.rs

```diff
@@ -96,12 +96,12 @@ impl<'a> Writer<'a> {
     /// If you write media before `IceConnectionState` is `Connected` it will be dropped.
     ///
     /// Panics if [`RtcConfig::set_rtp_mode()`][crate::RtcConfig::set_rtp_mode] is `true`.
-    pub fn write(
+    pub fn write<T: Into<Vec<u8>>>(
         self,
         pt: Pt,
         wallclock: Instant,
         rtp_time: MediaTime,
-        data: &[u8],
+        data: T,
```

Comment:
This can just be `impl Into<Vec<u8>>` which is slightly more ergonomic I think

Reply:
@k0nserv , @algesten ,

Fixed!